pages/Compressor.md

```diff
@@ -0,0 +1,13 @@
+The Compressor is in the [Basic Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machines) category. It is a multiblock structure.<br>
+
+## Obtaining
+The Compressor is a multiblock structure, it consists of a dispenser, a nether brick fence and 2 pistons.<br>
+
+A complete setup should look like this:
+
+![Compressor Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-compressor.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your compressed items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Compressor is used to compress items.
```

Comment:
This is a wiki, please be more precise than that.
State that it is used to craft various items and that they should look into the Slimefun Guide for more.